packages/gamut/src/ButtonBase/ButtonBase.tsx

```diff
@@ -59,35 +59,32 @@ const ResetElement = styled('button', styledOptions<'button'>())(resetStyles);
 export const ButtonBase = forwardRef<
   HTMLButtonElement | HTMLAnchorElement,
   any
->(
-  (
-    { href, disabled, children, role = 'button', type = 'button', ...rest },
-    ref
-  ) => {
-    if (href == null || disabled) {
-      return (
-        <ResetElement
-          {...rest}
-          ref={ref as MutableRefObject<HTMLButtonElement>}
-          as="button"
-          type={type}
-          role={role}
-          disabled={!!disabled}
-        >
-          {children}
-        </ResetElement>
-      );
-    }
-
+>(({ href, disabled, children, role, type = 'button', ...rest }, ref) => {
+  if (href == null || disabled) {
     return (
       <ResetElement
         {...rest}
-        ref={ref as MutableRefObject<HTMLAnchorElement>}
-        as="a"
-        href={href}
+        ref={ref as MutableRefObject<HTMLButtonElement>}
+        as="button"
+        type={type ?? 'button'}
```

Comment:
Nit: covered in props?

```suggestion
        type={type}
```

Reply:
we want it to default to `role="button"` only if it doesnt have an `href`, if its a link then role isnt necessary and this lets it be set to `undefined`

Reply:
Agreed - and `type = 'button'` above, so `type` is always going to be `string` on line 69.